sphinx-doc/code-evaluation.rst

```diff
@@ -68,3 +68,17 @@ To enable live updates, make sure Indium is set up to use local files (see
   current buffer.
 - ``(setq indium-update-script-on-save t)``: Automatically update the runtime
   script JavaScript source after each buffer save.
+
+You can setup a hook for the script update. For example ::
```

Comment:
What about rephrasing this sentence like this: `You can setup a hook to be run after each script update. For example ::`